conda_forge_tick/migrators/version.py

```diff
@@ -418,6 +431,12 @@ def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
                     >= VersionOrder(str(attrs["new_version"]))
                     for h in attrs.get("PRed", set())
                 )
+                # ...
+                or check_version_matches_cadence(
```

Comment:
I think you need a `not` here, my understanding is that we want to issue a PR if the pattern matches.

Reply:
do we need I True statment for the `filter_version` or a False for it to issue a PR (I thought that to perform the PR that should return a True value)

Reply:
False to issue PR, maybe I have it backwards

Reply:
I need to insert a `else` after the comparasion cause if `new_version >= version` it have to return a False (currently there is no response for that situation, and we only return false when `pattern == None`)